src/neural/loader.cc

```diff
@@ -140,6 +209,37 @@ void FixOlderWeightsFile(WeightsFile* file) {
       net->set_smolgen_activation(pblczero::NetworkFormat::ACTIVATION_SWISH);
     }
   }
+
+  // Get updated network format.
+  network_format = file->format().network_format().network();
+  auto embedding_type = file->format().network_format().input_embedding();
+  bool multihead_format = (network_format & 128) == 128;
```

Comment:
Make a helper function instead of `& 128`

Reply:
Hopefully this can be removed completely after we finalize net.proto.